addon/components/ui-calendar/component.ts

```diff
@@ -66,7 +68,7 @@ export default class UiCalendar extends Component {
       currentMonth = this.center.format("M");
     }
     let formattedSelected = this.selected.map((selected) =>
-      selected.format("YYYY-MM-DD")
+      moment(selected).format("YYYY-MM-DD")
```

Comment:
Shouldnt this already be a `moment`?

Reply:
The majority of the time yes, but in some cases it might not be, so I thought it's better to enforce that it is a `moment` rather than assume. This will also help when gradually moving away from moment in the long run